src/utilities/Helpers.js

```diff
@@ -12,6 +12,55 @@ var jsface = require('jsface'),
 var Helpers = jsface.Class({
 	$singleton: true,
 
+    validateConfigFile: function(file) {
+        if (!fs.existsSync(file)) {
+            Errors.terminateWithError("Please specify the config as a valid file");
+        }
+    },
+
+    validateAndReturnConfig: function(file) {
+        var validOptions = {
+                collection: "string",
+                url: "string",
+                folder: "string",
+                environment: "string",
+                exportEnvironment: "string",
+                data: "string",
+                global: "string",
+                exportGlobals: "number",
+                delay: "number",
+                requestTimeout: "number",
+                stopOnError: "boolean",
+                noSummary: "boolean",
+                noColor: "boolean",
+                insecure: "boolean",
+                tls: "boolean",
+                encoding: "string",
+                exitCode: "boolean",
+                outputFile: "string",
+                testReportFile: "string",
+                import: "string",
+                pretty: "boolean",
+                html: "string"
+            },
```

Comment:
Now that's one more place to maintain the list of config. Not that it's too much pain, but is there a way to pick this up straight from commander definition? :-p